### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by davidigbo**
## In this pull request, I implement the following things:

- [ ] Create class Person with the following:
- [ ] Instance vars: @id, @name, and @age.
- [ ] Constructor with name, age, and parent_permission as parameter. name and parent_permission are optional and have default values of "Unknown" and true.
- [ ] Getters for @id, @name, and @age.
- [ ] Setters for @name and @age.
- [ ] Private method of_age? that returns true if @age is greater or equal to 18 and false otherwise.
- [ ] Public method can_use_services? that returns true if person is of age or if they have permission from parents.
- [ ] Create class Student with the following:
- [ ] Inherits from Person.
- [ ] Constructor extends parent's constructor by adding @classroom and a parameter for it.
- [ ] Method play_hooky that returns "¯\(ツ)/¯".
- [ ] Create class Teacher with the following:
- [ ] Inherits from Person.
- [ ] Constructor extends parent's constructor by adding @specialization and a parameter for it.
- [ ] Override can_use_services? so it always returns true.
- [ ] Each class should be saved in a separate file.

### CODE REVIEW

File: README.Md [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+## ❓ FAQ (OPTIONAL) <a name="faq"></a>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](/LICENSE.md) licensed.
+
```
Comment (omar-muhamad) [inline README.Md:121]: - [ ] The MIT license link is broken, Please add the license file and check the link.

image

### Other than this you have done a great job💪.

**Review by omar-muhamad — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi @david30-maker,

### Nice job, you are almost there.

cat-driving-serious

## (Highlights) Good Points: :+1:
- Your work is well documented. ✔
- The correct workflow is used.  ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- [ ] Please check the inline comments below ⬇️.

## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by davidigbo** [comment]
Thanks @Omar-Muhamad for reviewing my project, I will implement the required changes.

**Review by m05e5 — APPROVED** [review]
**Hi @david30-maker**
You did great. Your project is complete!  There is nothing else to say other than... it's time to merge:shipit: 
Congratulations! 🎉 
CleverScornfulBuffalo-size_restricted

## Highlights


✔️ Clean Codebase
✔️ Great job implementing the requested changes

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**m05e5**_

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._**

**Comment by davidigbo** [comment]
Thanks @m05e5 for approving my project.

**Pull request merged**